src/Components/Members/MembersCreateEdit.js

```diff
@@ -0,0 +1,316 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import '../FormStyles.css';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import TextField from '@mui/material/TextField';
+import Card from '@mui/material/Card';
+import Button from '@mui/material/Button';
+import CardHeader from '@mui/material/CardHeader';
+import { Alert } from '@mui/material';
+import { useDropzone } from 'react-dropzone';
+
+const thumb = {
+  display: 'inline-flex',
+  borderRadius: 2,
+  border: '1px solid #eaeaea',
+  margin: 'auto',
+  width: 'auto',
+  height: 60,
+  padding: 4,
+  boxSizing: 'border-box',
+};
+
+function MembersCreateEdit() {
+  const { id } = useParams();
+  const urlEdit = `http://ongapi.alkemy.org/public/api/members/${id}`;
+  const urlCreate = `http://ongapi.alkemy.org/public/api/members`;
+  const [initialValues, setInitialValues] = useState({
+    name: '',
+    image: '',
+    description: '',
+    facebookUrl: '',
+    linkedinUrl: '',
+  });
+
+  const [files, setFiles] = useState([]);
+  const { getRootProps, getInputProps } = useDropzone({
+    accept: 'image/png, image/jpg',
+    onDrop: (acceptedFiles) => {
+      setInitialValues({
+        ...initialValues,
+        image: acceptedFiles.map((file) =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file),
+          }),
+        ),
+      });
```

Comment:
what is being done inside of setInitialValues, image? The answer of that question is a good candidate for the name of the function where this peace of code should be. Please move this to a separated function and then call it here.
```
const getImageWithUrl= (acceptedFiles) => acceptedFiles.map((file) =>
          Object.assign(file, {
            preview: URL.createObjectURL(file),
          }),
        );
```
then on the onFileDrop fuction
```
setInitialValues({...initialValue, image: getFilesWithUrl(acceptedFiles)})
```